entity-service/src/integrationTest/java/org/hypertrace/entity/service/service/EntityDataServiceTest.java

```diff
@@ -179,27 +179,17 @@ public void testCreateWithIdentifyingAttributes() {
 
   @Test
   public void testCreateV1EntityTypeWithoutIdentifyingAttributes() {
-    boolean upsertFailed = false;
-    Entity entity = Entity.newBuilder()
-        .setTenantId(TENANT_ID)
-        .setEntityType(EntityType.K8S_POD.name())
-        .setEntityName("Some Service")
-        .build();
-    try {
-      entityDataServiceClient.upsert(entity);
-    } catch (RuntimeException re) {
-      Throwable throwable = re.getCause();
-      assertTrue(throwable instanceof StatusRuntimeException);
-      StatusRuntimeException sre = (StatusRuntimeException) throwable;
-      upsertFailed = true;
-      assertEquals(Code.INTERNAL, sre.getStatus().getCode());
-      assertNotNull(sre.getStatus().getDescription());
-      assertTrue(sre.getStatus().getDescription().contains(
-          "expected identifying attributes"));
-    }
-    if (!upsertFailed) {
-      fail("Expecting upsert v1 typed entity to fail as identifying attributes are not set");
-    }
+    Entity entity =
+        Entity.newBuilder()
+              .setTenantId(TENANT_ID)
```

Comment:
very nit: is this 6 spaces?

Reply:
I think it's just aligning the dots 🤷  I just listen to the formatter, but after all the changes land I'll add spotless to this repo :)